src/libraries/Common/src/System/Text/ValueStringBuilder.cs

```diff
@@ -158,7 +158,12 @@ public void Insert(int index, string? s)
 
             int remaining = _pos - index;
             _chars.Slice(index, remaining).CopyTo(_chars.Slice(index + count));
-            s.AsSpan().CopyTo(_chars.Slice(index));
+#if SYSTEM_PRIVATE_CORELIB
```

Comment:
Presumably if https://github.com/dotnet/runtime/issues/51061 is approved we'd want this to be `NET6_OR_GREATER` instead?

Reply:
Yup (assuming it's used by any projects that target downlevel... if it's not, just removing the ifdef completely).